### ISSUE TITLE
Manual Windows Installation - Zim doesn't start

### ISSUE THREAD

**Issue by freyana** [zim-desktop-wiki/zim-desktop-wiki#1045]
Hi everyone

I have tried to install Zim on Windows, following Brendan Kidwell's instructions (https://www.glump.net/howto/desktop/install-zim-0-70-or-later-in-windows).

I followed the steps carefully and everything seemed fine. However, Zim does not run, nothing seems to happen, it just silently fails. 

I have double-checked every step of the installation, but cannot find any problem. My computer runs on a Windows 10, 64 Bit, German OS.

Would anyone have advice for me?
Or does anyone have advice, where I could find some logs or further information to do my own troubleshooting?

Thanks to all the developers and supporters for your constant effort :-)

Regards, Freyana

**Comment by d33pfr13d** [comment]
Hey everyone,

first of all: I love zim and have been using it for a long time now.

Got a new windows 10 computer and the same problem: Zim just does not start.
Using the "debug mode" starter I found the following error in the "zim.exe.log":

Traceback (most recent call last):
  File "zim.py", line 42, in <module>
  File "zim\__init__.pyo", line 147, in <module>
  File "locale.pyo", line 581, in setlocale
locale.Error: unsupported locale setting

Found some information on this error. Seems that in recent windows 10 versions python can't determine the system locale. Unfortunatley setting some enviroment variable (LC_ALL ?) does not help.

An easy fix would probably be to hardcode the locale the en_US in __init__.py.
I am a software developer myself, but not familiar with python and sofar can't get the source code to run with "msys2" as suggested.

Would it be possible to include a fallback on error to the english locale?

Thank you so much in advance.

**Comment by jaap-karssenberg** [comment]
You can set the locale using the LANG environment parameter. Hippe that
helps.



Op vr 10 apr. 2020 20:46 schreef d33pfr13d <notifications@github.com>:

**Comment by FloGMX** [comment]
Happy Easter,

the installation according to Brendan Kidwell instructions worked. Zim can be started.

However, the menu language always remains in English.
Regardless of whether I enter set LANG = de_DE in the CMD file described or I use the environment variable LANG de_DE. Only the language in the journal and in the Help / About menu item is then in the specified language. I've already tested several languages.

How can the debug mode be executed in Windows?

with friendly greetings

**Comment by FloGMX** [comment]
Hello freyana,

did you install the two programs msys and zim in the folder as described? In the programs folder (C: \ Programs) there are always problems with the write rights.

If you add another line with "pause" to the end of the created CMD file, the command window remains until another key is pressed. Look at the line with "Start" whether the directories and files are correct here?

**Comment by introt** [comment]
Hi @freyana, are you still having this issue?

**Comment by freyana** [comment]
Hi @introt. Thanks for reaching out to me. 

After asking this question, I kind of gave up and did not use Zim for quite some time. But I still missed it and so I came back to it to try again. I found the new Windows installer and was able to install and configure Zim. 

I am currently running version 0.73.5 which is working fine. A great thank you to the person who has created the Windows installer :-)

Thanks, Freyana
